eventmesh-connectors/eventmesh-connector-file/src/main/java/org/apache/eventmesh/connector/file/source/connector/FileSourceConnector.java

```diff
@@ -45,20 +56,24 @@ public Class<? extends Config> configClass() {
     public void init(Config config) throws Exception {
         // init config for hdfs source connector
         this.sourceConfig = (FileSourceConfig) config;
-
+        this.filePath = ((FileSourceConfig) config).getConnectorConfig().getFilePath();
+        this.fileName = getFileName(filePath);
     }
 
     @Override
     public void init(ConnectorContext connectorContext) throws Exception {
         SourceConnectorContext sourceConnectorContext = (SourceConnectorContext) connectorContext;
         this.sourceConfig = (FileSourceConfig) sourceConnectorContext.getSourceConfig();
-        this.offsetStorageReader = sourceConnectorContext.getOffsetStorageReader();
-
     }
 
     @Override
     public void start() throws Exception {
-
+        if (fileName == null || fileName.isEmpty() || filePath == null || filePath.isEmpty()) {
+            this.bufferedReader = new BufferedReader(new InputStreamReader(System.in, StandardCharsets.UTF_8));
+        } else {
+            this.bufferedReader = new BufferedReader(new InputStreamReader(Files.newInputStream(
+                Paths.get(filePath, fileName)), StandardCharsets.UTF_8), BUFFER_SIZE);
```

Comment:
In your new commit
1. I think there is no need to make separate judgments for `fileName` and `filePath` in this  conditional judgment `if (fileName == null || fileName.isEmpty() || filePath == null || filePath.isEmpty())`.
2. What will `Paths.get("/path/to/example.txt", "example.txt")` return?

I would appreciate it if you could test the functionality of this File Source Connector locally. I believe it is a very important job.

Reply:
Updated.Also I have tested the File Source Connector for **_start(), stop()_** and **_poll()_** method, locally.I am attaching the Ss of the tests I ran on this class below.  
![image](https://github.com/apache/eventmesh/assets/96522794/48879a11-8f69-44a9-8f12-74715002d110)
![image](https://github.com/apache/eventmesh/assets/96522794/15918ada-ab67-4167-a9db-86ffe4fd772a)